Resources/Textures/Objects/Misc/spaceshroom.rsi/meta.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": 1,
+    "license": "CC0-1.0",
+    "copyright": "GitHub @forthbridge",
```

Comment:
This should be more clear, like `Created by @forthbridge (GitHub)`

Reply:
Changed to `Created by @UbaserB (GitHub)`, who made the textures themselves